### PR TITLE
jboss-forge 2.20.1.Final

### PR DESCRIPTION
**Pull request by gastaldi**
Added `devel` block for the 3.0.0.Alpha1 release

### CODE REVIEW

File: Library/Formula/jboss-forge.rb [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
-  url "https://downloads.jboss.org/forge/releases/2.20.0.Final/forge-distribution-2.20.0.Final-offline.zip"
-  version "2.20.0.Final"
-  sha256 "ee4f9b2fff8ac72714cfd0c852735fcdd04e1b4462de8a6ba0807cbe3df4503f"
+  url "https://downloads.jboss.org/forge/releases/2.20.1.Final/forge-distribution-2.20.1.Final-offline.zip"
+  version "2.20.1.Final"
+  sha256 "dbe51b8c9cd0cf1b0a9adadcd8161c442a4ab586ca63f3a3f6d3369a2831afae"
 
   bottle :unneeded
-
+
```
Comment (bfontaine) [inline Library/Formula/jboss-forge.rb]: Please remove the trailing spaces on this line.

File: Library/Formula/jboss-forge.rb [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
+  sha256 "dbe51b8c9cd0cf1b0a9adadcd8161c442a4ab586ca63f3a3f6d3369a2831afae"
 
   bottle :unneeded
-
+  
+  devel do
+    url "https://downloads.jboss.org/forge/releases/3.0.0.Alpha1/forge-distribution-3.0.0.Alpha1-offline.zip"
+    sha256 "81d4ecafd3f555b81c139ab9c56ed79104714678bc47c5703ddb2875628afb1e"
+  end
+
```
Comment (bfontaine) [inline Library/Formula/jboss-forge.rb]: And here as well (trailing spaces).

File: Library/Formula/jboss-forge.rb [hunk 3]
```diff
@@ -1,12 +1,17 @@
...
-  sha256 "ee4f9b2fff8ac72714cfd0c852735fcdd04e1b4462de8a6ba0807cbe3df4503f"
+  url "https://downloads.jboss.org/forge/releases/2.20.1.Final/forge-distribution-2.20.1.Final-offline.zip"
+  version "2.20.1.Final"
+  sha256 "dbe51b8c9cd0cf1b0a9adadcd8161c442a4ab586ca63f3a3f6d3369a2831afae"
 
   bottle :unneeded
-
+  
+  devel do
+    url "https://downloads.jboss.org/forge/releases/3.0.0.Alpha1/forge-distribution-3.0.0.Alpha1-offline.zip"
```
Comment (bfontaine) [inline Library/Formula/jboss-forge.rb]: It seems that Homebrew can’t parse the version from this URL; please add `version "3.0.0.Alpha1"` below this line, similar to the `version` line at the top of the formula:

```rb
devel do
  url "..."
  version "..."
  sha256 "..."
end
```

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/0d661be6a313f8091f4c8e8bee78282ce0a2a5c9. Thank you for your contribution to Homebrew @gastaldi; we appreciate it! :smiley_cat:

**Pull request closed without merging**